common/include/pcl/common/vector_average.h

```diff
@@ -50,7 +50,7 @@ namespace pcl
     * \ingroup common
     */
   template <typename real, int dimension>
-  class VectorAverage 
+  class PCL_EXPORTS VectorAverage
```

Comment:
Why do we need `PCL_EXPORTS` here?

Reply:
Sorry, I was not sure about this. I quickly read the definition of `PCL_EXPORTS` and I think I saw that this macro also aligns the size of the class to 16bytes. 
What I wanted to do is use `EIGEN_ALIGN16` which is also done for many classes in PCL.
A colleague told me he had issues in his code, if he did not align the size of a class to 16bytes. Since the alignment operator is only used when the object is allocated.
Maybe this was a  misunderstanding. 

What do you think?
I will fix it as soon as possible.
Cheers

Reply:
Not really, `PCL_EXPORTS` evaluates to `__declspec(dllexport)` on Windows platform and to nothing everywhere else. As such, it only affects visibility of symbols in DLLs, nothing to do with alignment.